tests/test_sortedLinkedList.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+from ContractsFixture import ContractsFixture
+from ethereum.tester import TransactionFailed
+from pytest import raises
+
+def test_sortedLinkedList():
+    fixture = ContractsFixture()
```

Comment:
This class has been deleted on develop in the most recent merge, you'll need to switch over to the dependency injection pattern.